### DISCUSSION TITLE
How to enable dark mode for ChromiumWebBrowser?

### DISCUSSION THREAD

**Discussion by ElektroStudios** [cefsharp/CefSharp#4722] [Q&A] [answered]
I've opened this simple and detailed question in Stackoverflow, but I still can't get the help that I'm looking for...

 - [How to enable dark mode for CefSharp web browser in .NET?](https://stackoverflow.com/questions/78011208/how-to-enable-dark-mode-for-cefsharp-web-browser-in-net)

> My intention is that when loading a web page in the CEF-based browser, it will automatically be colored with a dark theme.
> 
> Or at least, and as a last resort, only a solution for websites that checks whether the web browser is using dark or light mode and then the website automatically adapts its colors.

_For .NET Core Windows Forms and .NET Framework Windows Forms._

Thanks in advance if someone could clear my doubts.

**Comment by amaitland** [comment, marked as answer]
See https://github.com/cefsharp/CefSharp/discussions/3955#discussioncomment-1944133 for details on the supported option.

In future, please just ask your question **ONCE**. An issue, a discussion and a post on stackoverflow isn't required, also creates extra work for me which I don't appreciate. I'd commented on stackoverflow, I just hadn't found time to follow up yet. Please just ask your question once and be patient.

**Reply by ElektroStudios** [reply]
I have tried to respect the code of conduct not by asking directly in the issues, but in the community. The fact that I published my doubt on an external website like stackoverflow.com I think it cannot be considered conflicting or relevant since it is an external website (I can't know that you also frequent that website).

But yes, you have reason, excuse me, because I recognize that I have tried to make my doubt visible and obtain the largest possible audience by asking here and there, and also by publishing a feature request related to the doubt which in fact its like asking the same question two times.

I did it because seeing how difficult it was for me to find a solution or find questions from other users with a solution for this question (or a solution that I could understand), I thought that my doubts would be doomed to not obtain a solution.

But I understand your point of view, it was my fault, excuse me.

And the help you've given me has been very surprising (as I said, I thought that my doubts would be doomed to not obtain a solution) and gratifying. You made it look so easy to solve.

Thanks a lot!
